src/main/rx/cyrf6936_spektrum.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define DSM_BIND_TIMEOUT_US 10000
+#define DSM_SYNC_TIMEOUT_US 20000
+#define DSM_RECV_LONG_TIMEOUT_US 18010
+#define DSM_RECV_MID_TIMEOUT_US 7000
+#define DSM_RECV_SHORT_TIMEOUT_US 4010
+#define DSM_RECV_TIMEOUT_OFFSET_US 1000
+
+#define DSM_MAX_MISSED_PACKETS 31
+
+#define DSM_MAX_RF_CHANNEL 0x4F
+#define DSM_MAX_CHANNEL_COUNT 12
+#define DSM_INITIAL_BIND_CHANNEL 0x0D
+
+#define DSM_TELEMETRY_FRAME_RPM 0x7E
+#define DSM_TELEMETRY_TIMEOUT_US 1500
+#define DSM_TELEMETRY_TIME_US 176000
+
+typedef enum {
```

Comment:
Are all of these type definitions and defines meant to be used outside of `cyrf6936_spektrum.c`? If not they should not be part of the header file / API

Reply:
No, I'll refactor them into cyrf6936_spektrum.c.

Reply:
Actually, I just started refactoring, but noticed that I use them in the unit test.

Reply:
I'd go for defining the ones that are needed in the unit test in the test itself - I don't think that things used for testing only should be part of the API.